src/extension-utilities.ts

```diff
@@ -103,4 +103,16 @@ export class ExtensionUtilities {
       }
     }
   }
+
+  static async handleError(err: unknown): Promise<void> {
+    let msg = 'unknown error';
+
```

Comment:
I will look more into this. I took out the `console.log` because sinon, has decided it does not like stubbing console methods anymore and it caused a bunch of failing tests, but I can use a wrapper method and test it that way.

Reply:
Digging into this more, it I don't think VS Code shows `console` methods in production. I couldn't find documentation clarifying, but I added some `console.error` calls in the extension.js and created a .vsix and installed it and they weren't logged anywhere, so it looks like there's no point in adding them.

Reply:
Good call. It looks like this issue https://github.com/Microsoft/vscode/issues/10063 confirms what you were seeing